spaceros/Earthfile

```diff
@@ -113,31 +119,38 @@ spaceros-artifacts:
   END
 
   IF [ $SPACEROS_REPO_URL = "https://github.com/space-ros/space-ros.git" ] 
-    # run wget because main repo's host is known + it is public 
+    # run wget instead of `GIT CLONE`, because main repo's host is known + it is public 
     RUN wget https://raw.githubusercontent.com/space-ros/space-ros/${_GIT_COMMIT_HASH}/ros2.repos
     RUN wget https://raw.githubusercontent.com/space-ros/space-ros/${_GIT_COMMIT_HASH}/excluded-pkgs.txt
   ELSE
-    GIT CLONE --branch ${_GIT_COMMIT_HASH} ${SPACEROS_REPO_URL} .
+    GIT CLONE --branch ${SPACEROS_GIT_REF} ${SPACEROS_REPO_URL} .
```

Comment:
During tests i've realized that commit hash is not supported in this git clone command. Earthly does not throw any error, but logs that there was some issue during cloning and proceeds further. I guess it then also reuses `ros2.repos` file from cache. Weird, but now it should be fixed.

Reply:
Yes, I tested it for tags as well and did not face any issues :)